de.lproj/ShortcutRecorder.strings

```diff
@@ -29,7 +29,7 @@
 "The key combination \"%@\" can't be used!" = "Die Tastenkombination „%@“ kann nicht verwendet werden.";
 
 /* No comment provided by engineer. */
-"Type shortcut" = "Kurztasten eingeben";
+"Type shortcut" = "Tastenkombination eingeben";
```

Comment:
Also, where this shows ( at least on Snagit ) the shorter the word the better.

Reply:
will do

Reply:
I let Claire know, and also sent her a screenshot of where it shows, because the same "shorter the better" applies to Camtasia too. the field isn't very wide